### ISSUE TITLE
Using statements inside namespace block cause incorrect formatting

### ISSUE THREAD

**Issue by humphs** [Projektanker/code-butler#3]
This code should be enough to reproduce. Running Code Butler inside VS Code

```
namespace Test
{
    using System.Threading.Tasks;

    public interface ITest
    {
        Task Test();
    }
}
```

Run: **Code Butler: Cleanup C# file**

File is formatted to:

```
namespace Test
{
    using System.Threading.Tasks;
    public interface ITest
    {
        Task Test();
    }
}
```

The newline between the using statements and the interface is removed incorrectly.

**Comment by just-seba** [comment]
Thank you for using Code Butler and opening this issue.

I will fix this bug and release a new version as soon as I can.

**Comment by just-seba** [comment]
Should be fixed with release of [version 1.2.0](https://marketplace.visualstudio.com/items?itemName=projektanker.code-butler)
Please let me know if there is anything else I can improve.
